### PR TITLE
provider/openstack: openstack_networking_port_v2 resource

### PR DESCRIPTION
**Pull request by jtopjian**

### CODE REVIEW

File: builtin/providers/openstack/resource_openstack_networking_network_v2_test.go [hunk 1]
```diff
@@ -104,6 +107,81 @@ func TestAccNetworkingV2Network_netstack(t *testing.T) {
...
+        ip_protocol = "tcp"
+        cidr = "0.0.0.0/0"
+      }
+    }
+
+    resource "openstack_networking_port_v2" "foo" {
+      region = "%s"
+      name = "port_1"
+      network_id = "${openstack_networking_network_v2.foo.id}"
+      admin_state_up = "true"
```
Comment (Fodoj) [inline builtin/providers/openstack/resource_openstack_networking_network_v2_test.go]: Would it make sense to also allow none-string `true`? I was surprised that it doesn't work with boolean value

File: website/source/docs/providers/openstack/r/networking_port_v2.html.markdown [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* `mac_address` - (Optional) Specify a specific MAC address for the port. Changing
+    this creates a new port.
+
+* `tenant_id` - (Optional) The owner of the Port. Required if admin wants
+    to create a port for another tenant. Changing this creates a new port.
+
+* `device_owner` - (Optional) The device owner of the Port. Changing this creates
+    a new port.
+
+* `security_groups` - (Optional) A list of security groups to apply to the port.
```
Comment (Fodoj) [inline website/source/docs/providers/openstack/r/networking_port_v2.html.markdown]: may be then `security_group_ids`?

Comment (jtopjian) [inline website/source/docs/providers/openstack/r/networking_port_v2.html.markdown]: Yes, this should be updated to more clearly reflect what is mentioned in the body of the text. I've made a note to open an issue when I have a second - or if you wanted to open one, that'd be great :smile:

File: builtin/providers/openstack/resource_openstack_networking_network_v2_test.go (hunk 1 above)
Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_networking_network_v2_test.go]: `admin_state_up` is defined as a String in the port resource, but boolean in all other network resources. Fixing this.

**Pull request merged**